README.md

```diff
@@ -46,6 +46,15 @@ redis:
 This is also where you will place any other customizations, such as API tokens
 or certificate paths.
 
+Config settings that vary in value depending on the deployment environment will need
+to be additionally  in the relevant [devops](https://github.com/department-of-veterans-affairs/devops/blob/master/ansible/deployment/config/vets-api) configs. Things such as:
+
+* API keys/tokens
```

Comment:
Maybe pull things together, to introduce the bullets?  Like:



~Things such as:~
(new paragraph)

Some examples of config that will need to be added to these files are:

- API keys/tokens
- 3rd party service hostnames
- Betamocks settings